### PR TITLE
finishing placing of pieces

### PR DESCRIPTION
**Pull request by przemyslawren**

### CODE REVIEW

File: src/Board.java [hunk 1]
```diff
@@ -52,12 +52,36 @@ private void PlacePieces(){
...
     private void PlaceWhitePieces(){
         var playerColor = PlayerColor.White;
-        placePiece(new Rook(playerColor, new Position(0, 0)));
-        //reszta figur
+
+        placePiece(new Rook(playerColor, new Position(1, 1)));
+        placePiece(new Knight(playerColor, new Position(2, 1)));
+        placePiece(new Bishop(playerColor, new Position(3, 1)));
+        placePiece(new Queen(playerColor, new Position(4, 1)));
+        placePiece(new Knight(playerColor, new Position(5, 1)));
```
Comment (hubertniewinski) [inline src/Board.java]: king

File: src/Board.java [hunk 2]
```diff
@@ -52,12 +52,36 @@ private void PlacePieces(){
...
 
     private void PlaceBlackPieces(){
-        //analogicznie czarne
+        var playerColor = playerColor.Black;
+
+        placePiece(new Rook(playerColor, new Position(1, 8)));
+        placePiece(new Knight(playerColor, new Position(2, 8)));
+        placePiece(new Bishop(playerColor, new Position(3, 8)));
+        placePiece(new Queen(playerColor, new Position(4, 8)));
+        placePiece(new Knight(playerColor, new Position(5, 8)));
```
Comment (hubertniewinski) [inline src/Board.java]: king

File: src/Board.java (hunk 1 above)
Comment (przemyslawren) [reply src/Board.java]: done

File: src/Board.java (hunk 2 above)
Comment (przemyslawren) [reply src/Board.java]: done

**Pull request merged**